src/SFA.DAS.Reservations.Web/Views/EmployerReservations/SelectLegalEntity.cshtml

```diff
@@ -0,0 +1,98 @@
+﻿@using System.Text
+@using Microsoft.AspNetCore.Mvc.ModelBinding
+@using SFA.DAS.Reservations.Web.Infrastructure
+@model SelectLegalEntityViewModel
+@{
+    ViewData["Title"] = "Reservations - Select Organisation";
+}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        @if (!ViewData.ModelState.IsValid)
+        {
+            <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-title">
+                    There is a problem
+                </h2>
+                <div class="govuk-error-summary__body">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        @{
+                            var errorStringBuilder = new StringBuilder();
+                            var errorCount = ViewData.ModelState.ErrorCount;
+                            var loopCount = 1;
+                            foreach (var error in ViewData.ModelState)
+                            {
+                                if (error.Value.ValidationState == ModelValidationState.Invalid)
+                                {
+                                    errorStringBuilder.Append($"<li><a href=\"#{error.Key}\">TODO: what is this error message?</a></li>");
```

Comment:
Lets get this content in - you'll also need to look at getting the Id of the legal entity so it can be auto selected from the error link